mod_reforged/hooks/config/character_injuries.nut

```diff
@@ -1,74 +1,126 @@
-// Regular Damage Types
-::Const.Injury.BluntHead.extend([
-	{
-		ID = "injury.rf_black_eye",
-		Threshold = 0.25,
-		Script = "injury/rf_black_eye_injury"
-	},
-	{
-		ID = "injury.rf_dislocated_jaw",
-		Threshold = 0.25,
-		Script = "injury/rf_dislocated_jaw_injury"
-	}
-]);
+{	// Regular Injury Types
+	::Const.Injury.BluntHead.extend([
+		{
+			ID = "injury.rf_black_eye",
+			Threshold = 0.25,
+			Script = "injury/rf_black_eye_injury"
+		},
+		{
+			ID = "injury.rf_dislocated_jaw",
+			Threshold = 0.25,
+			Script = "injury/rf_dislocated_jaw_injury"
+		}
+	]);
 
-::Const.Injury.BurningBody.extend([
-	{
-		ID = "injury.rf_heat_stroke",
-		Threshold = 0.25,
-		Script = "injury/rf_heat_stroke_injury"
-	}
-]);
+	::Const.Injury.BurningBody.extend([
+		{
+			ID = "injury.rf_heat_stroke",
+			Threshold = 0.25,
+			Script = "injury/rf_heat_stroke_injury"
+		}
+	]);
+}
 
-// Event Container
-::Const.Injury.Brawl.extend([
-	{
-		ID = "injury.rf_black_eye",
-		Threshold = 0.25,
-		Script = "injury/rf_black_eye_injury"
-	},
-	{
-		ID = "injury.rf_dislocated_jaw",
-		Threshold = 0.25,
-		Script = "injury/rf_dislocated_jaw_injury"
-	}
-]);
+{	// Super-Categories
+	::Const.Injury.All.extend([
+		{
+			ID = "injury.rf_black_eye",
+			Threshold = 0.25,
+			Script = "injury/rf_black_eye_injury"
+		},
+		{
+			ID = "injury.rf_dislocated_jaw",
+			Threshold = 0.25,
+			Script = "injury/rf_dislocated_jaw_injury"
+		},
+		{
+			ID = "injury.rf_heat_stroke",
+			Threshold = 0.25,
+			Script = "injury/rf_heat_stroke_injury"
+		},
```

Comment:
Comma should be removed

Reply:
I am aware that commas are optional in squirrel (it will work even without commas in the entire array/table and with extra comma at the end). But your point that having commas at the end can lead to cleaner diff in commits is interesting. I don't know what the convention is in this case that we want to go with, so I'd ask Enduriel and if he's ok with this then we can do it.

Reply:
Based on the discussion in Discord we decided that we like these commas at the end. So you can disregard this comment.